aomp-amdgpu/PKGBUILD

```diff
@@ -1,23 +1,53 @@
-# Maintainer: acxz <akashpatel2008 at yahoo dot com>
+# Maintainer: Torsten Keßler <t dot kessler at posteo dot de>
+# Contributor: acxz <akashpatel2008 at yahoo dot com>
+
 pkgname=aomp-amdgpu
-pkgver=3.5.0
+pkgdesc='Clang/LLVM based compiler with added support for the OpenMP API on Radeon GPUs'
+_pkgver=11.6-2
+pkgver=11.6.2
 pkgrel=1
-pkgdesc="Clang/LLVM based compiler with added support for the OpenMP API on
-Radeon GPUs"
 arch=('x86_64')
-url="https://github.com/ROCm-Developer-Tools/aomp"
-license=('Apache-2.0')
-depends=('pciutils' 'numactl' 'libelf' 'libffi' 'python' 'openmpi')
-makedepends=('cmake')
-source=("$pkgname-$pkgver.tar.gz::https://github.com/ROCm-Developer-Tools/aomp/archive/rocm-$pkgver.tar.gz")
-sha256sums=('e4526489833896bbc47ba865e0d115fab278ce269789a8c99a97f444595f5f6a')
-
-build() {
-  cd ${srcdir}/aomp-rocm-$pkgver
-  nohup make
+url='https://github.com/ROCm-Developer-Tools/aomp'
+license=('Apache')
+depends=(z3 numactl pciutils libelf libffi)
+makedepends=(git cmake python)
+source=("${pkgname}-${pkgver}.tar.gz::$url/archive/rel_$_pkgver.tar.gz"
+        'disable_ocl_tests.patch'
+        'adjust_rpath.patch')
```

Comment:
I believe these patches are not included in the commit. Also if they can be contributed upstream that would be ideal.

Reply:
I've added the missing patch files. The first patch removes unnecessary tests, the second changes RPATH from `installdir`, which is `$pkdir/opt/rocm/aomp`, to `/opt/rocm/aomp`. This is only needed for packaging.

Reply:
I think the patches were added in the wrong directory.

I see kk

Reply:
Thanks. I've fixed it.